packages/editor-ui/src/components/CredentialsInput.vue

```diff
@@ -1,36 +1,36 @@
 <template>
 	<div @keydown.stop class="credentials-input-wrapper">
-		<el-row>
+		<el-row class="credential-name-wrapper">
 			<el-col :span="6" class="headline-regular">
 				Credentials Name:
-				<el-tooltip class="credentials-info" placement="top" effect="light">
+				<n8n-tooltip class="credentials-info" placement="top" >
 					<div slot="content" v-html="helpTexts.credentialsName"></div>
 					<font-awesome-icon icon="question-circle" />
-				</el-tooltip>
+				</n8n-tooltip>
 			</el-col>
 			<el-col :span="18">
-				<el-input size="small" type="text" v-model="name"></el-input>
+				<n8n-input size="medium" type="text" v-model="name"></n8n-input>
```

Comment:
let's have the `v-model` as first attribute

Reply:
changed